src/treemodel/delta/mergedelta.js

```diff
@@ -19,7 +20,35 @@ import CKEditorError from '../../ckeditorerror.js';
  *
  * @class treeModel.delta.MergeDelta
  */
-export default class MergeDelta extends Delta {}
+export default class MergeDelta extends Delta {
+	get moveOperation() {
+		return this.operations[ 0 ] || null;
+	}
+
+	get removeOperation() {
```

Comment:
I would be cool if deltas encapsulate informations about operations and return only meta informations about the delta (taken from operations). For example instead of: `delta.howManyMoved` instead of `delta.moveOperation.howMany`.